modules/030-cloud-provider-yandex/cloud-instance-manager/machine-class.yaml

```diff
@@ -48,7 +52,7 @@ spec:
   {{- $effectiveLabels = mergeOverwrite $effectiveLabels .nodeGroup.instanceClass.additionalLabels  }}
 {{- end }}
 {{- range $k, $v := $effectiveLabels }}
-    {{ $k }}: {{ $v }}
+    {{ $k | trunc 63 }}: {{ $v | trunc 63 }}
```

Comment:
i think we need more complex logic here - https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
key name must be less than 64 characters, but key with / and prefix can be 256 character length

Reply:
It's a restriction from yandex. I just restricted the same thing as above.